src/dlgPackageExporter.cpp

```diff
@@ -662,228 +665,231 @@ void dlgPackageExporter::slot_export_package()
         }
     }
 
+    qDebug() << stagingDirName << mPackagePathFileName << mXmlPathFileName << mPackageName << mPackageConfig << isOk;
+
     if (isOk) {
-        // zip error code:
-        int ze = 0;
-/*
-* Relevant options:
-* * ZIP_CREATE creates the archive if it does not exist.
-* * ZIP_TRUNCATE zaps any contents in a previously existing file.
-*/
-        // The pre-libzip 0.11 with the now obsolete zip_add() lacked the option
-        // of the later zip_file_add() which DOES have an overwrite option:
-        zip* archive = zip_open(mPackagePathFileName.toUtf8().constData(), ZIP_CREATE|ZIP_TRUNCATE, &ze);
-
-        if (!archive) {
-            // Failed to open/create archive file
-            // We now use the better error handling system (not requiring a
-            // previously defined finite-sized char type buffer {which obviously
-            // could have string buffer overflow issues} which is available in
-            // post 0.10 versions of libzip):
-            zip_error_t error;
-            zip_error_init_with_code(&error, ze);
-            displayResultMessage(tr("Failed to open package file. Error is: \"%1\".",
-                                    // Intentional comment to separate arguments
-                                    "This error message is shown when the libzip library code is unable to open the file that was to be the end result of the export process. As this may be an existing file anywhere in the computer's file-system(s) it is possible that permissions on the directory or an existing file that is to be overwritten may be a source of problems here.")
-                                 .arg(zip_error_strerror(&error)), false);
-            zip_error_fini(&error);
-            isOk = false;
-            // The above flag will now cause execution to drop down to the bottom of
-            // the method and set up a means to close the dialogue after the user
-            // has seen the error message...
+        mWritingZip = true;
+        slot_enableExportButton({});
```

Comment:
Having this here AND in the lambda function five lines or so down doesn't make sense - I suspect originally there was process text placed on the button indicating what was happening so that this one might be:
```suggestion
        slot_enableExportButton(tr("Exporting..."));
```

Reply:
There is already an `Exporting...` text, so I'd rather not duplicate it

Reply:
As it is, that slot throws away the argument - so it would have to be tweaked anyhow.... but it was more me trying to show how the text could be revised. Mind you in my past work in that area I think I made the button a one-shot (it could be clicked and then it would stay down and get disabled) so using it to show a brief indication of success or failure made sense then - it isn't clear to me as I type this whether a re-export is possible from a single run of this class.